data/members/core_members.json

```diff
@@ -0,0 +1,45 @@
+[
+    {
+        "first_name": "Wolfgang",
```

Comment:
Core members are a subset of collaboration members. Why are they a separate JSON?